src/Definitions/CSSDefinition.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Stevebauman\Purify\Definitions;
+
+use HTMLPurifier_CSSDefinition;
+
+interface CSSDefinition
```

Comment:
Can we adjust this interface name to be `PascalCase` for consistency since we have `HtmlDefinition` and not `Html5Definition`?

```suggestion
interface CssDefinition
```

Reply:
This should be updated now to PascalCase, thanks

Reply:
Awesome thanks @EPGDigital-MW! 🙏